Data-Transfer-ReactJS-Dashboard/src/pages/Layout.js

```diff
@@ -0,0 +1,27 @@
+import { Outlet, Link} from "react-router-dom";
+import './Layout.css';
+
+const Layout = () => {
+  return (
+    <>
+        <ul>
+          <li>
+            <Link to="/">Data Transfer Progress</Link>
+          </li>
+          <li>
+            <Link to="/healofdata">Health of Data</Link>
+          </li>
+          <li>
+            <Link to="/technicaldetails">Technical Details</Link>
+          </li>
+          <li>
+            <Link to="/transferqueue">Transfer Queue</Link>
+          </li>
+        </ul>
+
+      <Outlet />
+    </>
+  )
+};
```

Comment:
General comment: please be consistent with number of spaces.

Reply:
updated entire code two with two space indent.